ui/StatusQ/src/StatusQ/Core/Theme/Theme.qml

```diff
@@ -196,8 +196,10 @@ QtObject {
     readonly property real pressedOpacity: 0.7
 
     property int dynamicFontUnits: 0
+    property int currentFontSize: Theme.FontSizeM
```

Comment:
ideally that property (and also `dynamicFontUnits` above) should  should be read-only, with mutable counterparts hidden in `d`.